includes/class-plugin.php

```diff
@@ -665,15 +665,19 @@ public function render_admin_bar( $wp_admin_bar ) {
             return;
         }
 
+        $redis_status = $this->get_redis_status();
+
         $wp_admin_bar->add_node(
             [
                 'id' => 'redis-cache',
                 'title' => __( 'Object Cache', 'redis-cache' ),
+                'meta' => [
+                    'html' => '<style>#wpadminbar ul li.redis-cache-error { background: #c00; } #wpadminbar:not(.mobile) .ab-top-menu>li.redis-cache-error:hover>.ab-item { background: #b30000; color: #fff; }</style>',
```

Comment:
@tillkruss this isn't elegant, but it's the only way we can color the node without loading an external CSS file. We could inject the style into the head, but there doesn't seem any benefit.

Reply:
It seems efficient.

Reply:
It does 🫠